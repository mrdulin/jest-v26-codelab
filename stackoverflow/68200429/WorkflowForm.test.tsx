import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import WorkflowForm from './WorkflowForm';
import { cleanup } from '@testing-library/react';
import configureMockStore from 'redux-mock-store';

afterEach(cleanup);

const mockStore = configureMockStore();
const store = mockStore({
  data: { userID: '121' },
  ui: { userObj: 'Name' },
});

describe('Test suits for WorkflowForm', () => {
  it('should match with snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <BrowserRouter>
            <WorkflowForm />
          </BrowserRouter>
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
