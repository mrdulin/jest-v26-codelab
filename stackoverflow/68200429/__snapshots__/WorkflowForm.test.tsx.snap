// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for WorkflowForm should match with snapshot 1`] = `
<div>
  WorkflowForm
</div>
`;
